CONTRIBUTING.md

```diff
@@ -40,28 +55,44 @@ There are two types of automated tests:
 
 Both the unit and functional tests can be run together via:
 
-`./scripts/test.sh`
+`composer test`
 
 ### Unit Tests
 
 The unit test files are in the `tests/unit-tests` directory.
 
-To run the unit tests, simply execute:
+To run the unit tests for Terminus 0.x, simply execute:
 
   `vendor/bin/phpunit`
+  
+The Terminus 1.x unit tests can be run via:
+
+  ```bash
+  cd /install/location/terminus
+  composer phpunit
+  ```
+
 
 ### Functional Tests
 
-The functional test files are in the `test/features` directory. To run the entire test suite:
+The functional test files are in the `test/features` directory. To run the entire test suite for Terminus 0.x:
```

Comment:
I added a commit to explicitly provide the location of the Behat tests for the new code branch.